presto-hive/src/main/java/com/facebook/presto/hive/PartitionUpdate.java

```diff
@@ -205,4 +206,60 @@ public HiveBasicStatistics getStatistics()
         APPEND,
         OVERWRITE,
     }
+
+    public static class FileWriteInfo
+    {
+        private final String writeFileName;
+        private final String targetFileName;
```

Comment:
I still feel we want to make one of them optional... We don't have to do this now but maybe add a TODO?

Reply:
Let's discuss this in https://github.com/prestodb/presto/pull/12386#discussion_r263122837 as well :)